.github/workflows/build-http-broker-containers.yaml

```diff
@@ -0,0 +1,73 @@
+name: "Build HTTP Protocol Brokers"
```

Comment:
Do we need this workflow if it is in its own branch as a replacement for nessie? I am not sure if we want to provide akri http-brokers. Or maybe this is a good example of how a workflow file should come with a new discovery handler / protocol?

Reply:
i suggested it as a way to simplify people using the sample.  though i'm not entirely sure whether we ought to or not.  the pro's are that it would be simpler to pick up and try this out ... the cons are that we'd end up supporting more containers/labels.